exporter/azure_monitor/common/version.go

```diff
@@ -0,0 +1,4 @@
+package common
+
+var Opencensus_version = "0.3.dev0"

```

Comment:
Don't know if this makes sense, can we get the real opencensus-go sdk version?

Reply:
Sure, I found `opencensus.Version()` which seems to do the trick (https://github.com/census-instrumentation/opencensus-go/blob/master/opencensus.go).